docs/conf.py

```diff
@@ -63,7 +63,7 @@
 # built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = '1.1.3.dev0'
+release = '0.1.0.dev0'
```

Comment:
Is this right? It appears to be a global version.

Reply:
It is global, but I wasn't going to worry about it until we are ready to release.

We also need to figure out a better way to handle it, but not this PR.